samples/javascript/chatapp-aad/server.js

```diff
@@ -1,12 +1,27 @@
 const express = require('express');
 const { WebPubSubServiceClient } = require('@azure/web-pubsub');
 const { WebPubSubEventHandler } = require('@azure/web-pubsub-express');
-const { DefaultAzureCredential } = require('@azure/identity')
+const { DefaultAzureCredential, ClientSecretCredential, VisualStudioCodeCredential, AzureCliCredential, ClientCertificateCredential } = require('@azure/identity')
 
 const app = express();
 const hubName = 'chat';
 
-let serviceClient = new WebPubSubServiceClient(process.argv[2], new DefaultAzureCredential(), hubName);
+let endpoint = process.argv[2] // sample: https://<name>.webpubsub.azure.com
```

Comment:
`process.argv[2] || process.env.WebPubSubConnectionString;`

Reply:
This is `endpoint`, not `connection string`.